app/static/js/tickets/prepay-tickets.js

```diff
@@ -0,0 +1,19 @@
+$('.autofill-checkbox').click(function(){
```

Comment:
There is already a js file for this page `js/guest/ticketing.js`. put the code in that.